pyresample/spherical.py

```diff
@@ -26,6 +26,19 @@
 
 logger = logging.getLogger(__name__)
 
+EPSILON = 0.0000001
+
+
+def unwrap_radians(val, mod=np.pi):
+    """Put *val* between -*mod* and *mod*."""
+    val = np.asarray(val)
```

Comment:
Do we need this here? What are the cases where this isn't a numpy array?

Reply:
In `spherical.py`, `val` is either a float or a `np.array `. But if I provide a tuple of longitudes as input, then it would fail. Should I remove it @djhoese ?

Reply:
Is there code in `spherical.py` that passes a tuple? If your code only ever passes a scalar float or a numpy array then I'd say remove this. You could go even further and change the name of this to `_unwrap_radians` to identify to the reader of the code (in the future) that this function shouldn't be used outside of this module.